lib/http/fetch.js

```diff
@@ -1,7 +1,18 @@
 'use strict'
 
+const http = require('http')
+const https = require('https')
 const {hasBody} = require('type-is')
-const request = require('../util/request')
+const got = require('got')
+
+const agents = {
+  http: new http.Agent({
+    maxSockets: 4
```

Comment:
Should be configureable

Reply:
I’ll allow passing a custom agent through the configuration. It should be global.